### PR TITLE
Alter Options page based on Figma

### PR DESCRIPTION
**Pull request by wirednkod**
The main purpose of this pr is to adjust to new figma by @Goranch3 - again #980)
This PR partially satisfies the bug list on #1074 

Rest of bugs will be corrected in a different PR.

### CODE REVIEW

**Comment by wirednkod** [comment]
@tomaka I know this is a react PR but maybe you are interested to have a look on how `last block` is used and shown on [this commit.](https://github.com/paritytech/substrate-connect/pull/1082/commits/bbfc5b052a932a4be873ccf75f6186408680db6e)

**Comment by tomaka** [comment]
You can't just assume that `chain_newHead` notifications will be generated, you also need to subscribe to blocks.

File: projects/extension/src/background/ConnectionManagerWithHealth.ts [hunk 1]
```diff
@@ -50,6 +50,11 @@ export interface ChainInfo<SandboxId> {
    * The identifier for the sandbox that has received the message that requests to add a chain.
    */
   sandboxId: SandboxId
+
+  /**
+   * Last block of the chain
+   */
+  lastBlock: number
```
Comment (tomaka) [inline projects/extension/src/background/ConnectionManagerWithHealth.ts:57]:
```suggestion
   * Height of the current best block of the chain
   */
  latestBestBlock: number
```

"last" implies that there won't be any other block in the future.

File: projects/extension/src/components/IconWeb3.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import "../main.css"
+
+interface Props {
+  size?: string
+  color?: string
+  children?: string
+}
+
+const hasGlyph = (string: string) =>
+  ["kusama", "polkadot", "westend", "kulupu", "rococo"].indexOf(string) > -1
```
Comment (tomaka) [inline projects/extension/src/components/IconWeb3.tsx:11]:
```suggestion
const hasGlyph = (string: string) =>
  ["kusama", "polkadot", "westend", "rococo"].indexOf(string) > -1
```

Kulupu is so random here. There are dozens of chains more important than them.

Comment (wirednkod) [reply projects/extension/src/components/IconWeb3.tsx:11]: These are the chains supported at the moment from our "font" library.

File: projects/extension/src/background/ConnectionManagerWithHealth.ts [hunk 3]
```diff
@@ -427,6 +428,10 @@ export class ConnectionManagerWithHealth<SandboxId> {
                   break
                 }
               }
+            } else if (jsonRpcMessage.method === "chain_newHead") {
+              chain.bestBlockHeight = parseInt(
+                jsonRpcMessage.params.result.number,
+              )
```
Comment (tomaka) [inline projects/extension/src/background/ConnectionManagerWithHealth.ts:434]:
```suggestion
```

File: projects/extension/src/background/ConnectionManagerWithHealth.ts [hunk 4]
```diff
@@ -157,6 +157,7 @@ export class ConnectionManagerWithHealth<SandboxId> {
       return {
         peers: chain.peers,
         isSyncing: chain.isSyncing,
+        bestBlockHeight: chain.bestBlockHeight,
```
Comment (tomaka) [inline projects/extension/src/background/ConnectionManagerWithHealth.ts:160]: :+1:

File: projects/extension/src/background/ConnectionManagerWithHealth.ts [hunk 5]
```diff
@@ -59,7 +59,7 @@ export interface ChainInfo<SandboxId> {
    * This value should be treated as a hint, and not a strong information. It is possible for
    * the value to be erroneous, for example for chains that have tweaked their header format.
    */
-  latestBestBlockHeight?: number
+  bestBlockHeight?: number
```
Comment (tomaka) [inline projects/extension/src/background/ConnectionManagerWithHealth.ts:62]: Why the rename?

**Review by tomaka — CHANGES_REQUESTED** [review]

File: projects/extension/src/background/ConnectionManagerWithHealth.ts (hunk 3 above)
Comment (wirednkod) [reply projects/extension/src/background/ConnectionManagerWithHealth.ts:434]: What is the change that is suggested here?

File: projects/extension/src/background/ConnectionManagerWithHealth.ts (hunk 5 above)
Comment (wirednkod) [reply projects/extension/src/background/ConnectionManagerWithHealth.ts:62]: It felt only right to keep naming consistent across and I preferred the shorter-length name

File: projects/extension/src/background/ConnectionManagerWithHealth.ts (hunk 3 above)
Comment (tomaka) [reply projects/extension/src/background/ConnectionManagerWithHealth.ts:434]: Well, to remove this entirely. It was your first approach, which wasn't good.

File: projects/extension/src/background/ConnectionManagerWithHealth.ts (hunk 5 above)
Comment (tomaka) [reply projects/extension/src/background/ConnectionManagerWithHealth.ts:62]: I don't mind

Comment (wirednkod) [reply projects/extension/src/background/ConnectionManagerWithHealth.ts:62]: I'll keep it as is then

**Review by tomaka — APPROVED** [review]

**Pull request merged**